packages/react/src/components/ChatInput/AudioMessageRecorder.js

```diff
@@ -135,7 +135,7 @@ const AudioMessageRecorder = () => {
     if (file) {
       setFile(null);
     }
-  }, [isRecorded, file]);
+  }, [isRecorded, file, ECOptions.enableThreads, RCInstance, threadId]);
```

Comment:
Are you sure about this? There are some linting warnings for which a quick fix of adding dependencies actually messes things up and creates issues. If you are sure then fine

Reply:
I have faced such issues. That's why just letting you know...

Reply:
I tried to check after adding these as dependencies. Everything was working fine I guess.
Still I'll once more look into to be sure.

Thanks for pointing this out.

Reply:
I would suggest keeping the linting warnings if you are not sure about it. Linting warnings won't create any issues while committing.

Reply:
Yup agreed with @Spiral-Memory , can you please check @umangutkarsh if these are actually required and if its completely safe to go ahead with/without them

Reply:
Hey @sidmohanty11 @Spiral-Memory . I've reverted and removed them from the dependencies so that nothing breaks.

Also one thing, after pulling the latest changes from the recent commits. The 'send-message' button is not working as expected. 
<img width="946" alt="Screenshot 2024-03-11 235358" src="https://github.com/RocketChat/EmbeddedChat/assets/95426993/b3c6dda5-6bc9-47b1-aab7-5c238abd1243">

It's opening this modal. Just wanted to confirm if this is something added or an issue.
Please let me know. Thanks.

Reply:
Thank you so much, @umangutkarsh , for the notice. Yes, this was a new feature introduced to handle excessively long messages, but it wasn't intended to work this way. Yesterday, I made some changes to the `sendMessage` function and added a parameter there, which defaults to false, to check whether it is an attachment. By pressing Enter, everything was working fine. However, when using the send-btn, I was mistakenly assigning that function rather than calling it like `()=>sendMessage()`. That's the reason this issue originated. 

Thank you so much for the notice. I have just fixed it in #510. @sidmohanty11 , please review it.